src/OpenDiffix.Core/Analyzer.fs

```diff
@@ -1,5 +1,7 @@
 module rec OpenDiffix.Core.Analyzer
 
+open System.Text.RegularExpressions
+
 open AnalyzerTypes
 open NodeUtils
```

Comment:
AFAIK we don't separate stdlib like in pg_diffix here. We start with `System.*` then others.